rxandroid/build.gradle

```diff
@@ -26,3 +41,76 @@ dependencies {
     testCompile 'org.mockito:mockito-core:1.10.19'
     testCompile 'org.robolectric:robolectric:2.4'
 }
+
+// Create source/javadoc artifacts for publishing
+apply from: "$rootDir/gradle/artifacts.gradle"
+
+// Configure android-maven-gradle-plugin
+install {
+    repositories.mavenInstaller.pom.project {
+        name POM_NAME
+        description POM_DESCRIPTION
+        packaging POM_PACKAGING
+        url POM_URL
+
+        licenses {
+            license {
+                name POM_LICENCE_NAME
+                url POM_LICENCE_URL
+                distribution POM_LICENCE_DIST
+            }
+        }
+
+        scm {
+            url POM_SCM_URL
+            connection POM_SCM_CONNECTION
+            developerConnection POM_SCM_DEV_CONNECTION
+        }
+
+        developers {
+            developer {
+                id 'JakeWharton'
+                name 'Jake Wharton'
+            }
+            developer {
+                id 'mttkay'
+                name 'Matthias Käppler'
+            }
+            developer {
+                id 'dlew'
+                name 'Dan Lew'
+            }
+        }
+    }
+}
+
+// Configure gradle-bintray-plugin
+bintray {
+    configurations = ['archives']
+
+    // TODO: DELETE ME WHEN READY!
+    dryRun = true
+
+    pkg {
+        repo = 'RxJava'
```

Comment:
As far as I could tell, this was the data sent by the rxjava nebula plugin: https://github.com/nebula-plugins/gradle-rxjava-project-plugin/blob/gradle-2.2/src/main/groovy/nebula/plugin/rxjavaproject/RxJavaPublishingPlugin.groovy